### ISSUE TITLE
World in Forge where this module was already installed are still broken after 0.2.6

### ISSUE THREAD

**Issue by Paulo-nf** [baileymh/statblock-library#18]
I did create a new world and enabled the module and it worked there. But my old world where it has been installed for a while remains broken.

**Comment by baileymh** [comment]
Have you tried uninstalling and reinstalling since the latest update?

**Comment by Paulo-nf** [comment]
Yes. I have just been using a different world where the module works and copying the statblock needed and then pasting it on the statblock converter.

**Comment by baileymh** [comment]
Try running `game.pf1.migrateWorld()` in  your main world and see if that fixes things.

**Comment by Paulo-nf** [comment]
Unfortunately no.

**Comment by baileymh** [comment]
Do any errors show up in the console when you try to import?

**Comment by Paulo-nf** [comment]
![Capture](https://user-images.githubusercontent.com/98553013/173208041-d346645e-a94c-49ac-9caf-4e92e9c0a9a2.PNG)
Similar error every time I open one of the compendiums. Honestly I worry that importing will break something.
